definitions/divorce/json/AuthorisationCaseField/AuthorisationCaseField-resp-journey-nonprod.json

```diff
@@ -18,7 +18,7 @@
     "CaseTypeID": "DIVORCE",
     "CaseFieldID": "RespondentOrganisationPolicy",
     "UserRole": "caseworker-divorce-solicitor",
-    "CRUD": "CRU"
+    "CRUD": "CRUD"
```

Comment:
It may be needed by https://github.com/hmcts/div-case-orchestration-service/pull/1354